HabitTracker/HabitTracker/Program.cs

```diff
@@ -0,0 +1,265 @@
+                connection.Open();
+                var tableCmd = connection.CreateCommand();
+
+                tableCmd.CommandText =
+                    $"SELECT * FROM drinking_water";
+
+                List<DrinkingWater> tableData = new();
+                var reader = tableCmd.ExecuteReader();
+
+                if (reader.HasRows)
+                {
+                    while (reader.Read())
+                    {
+                        tableData.Add(
+                            new DrinkingWater
+                            {
+                                Id = reader.GetInt32(0),
+                                Date = DateTime.ParseExact(reader.GetString(1), "dd-mm-yyyy", new CultureInfo("pl-PL")),
+                                Quantity = reader.GetInt32(2),
+                            });
+                    }
+
+                    connection.Close();
+
+                    Console.WriteLine("..............................................");
+                    Console.WriteLine("Records found:");
+                    foreach (var dw in tableData)
+                    {
+                        Console.WriteLine(
+                            $"ID: {dw.Id} - Date: {dw.Date.ToString("dd-mm-yyyy")} - Quantity: {dw.Quantity}");
+                    }
+
+                    Console.WriteLine("..............................................");
+                }
+                
+                else Console.WriteLine("No rows found");
+            }
+        }
+
+        void InsertRecord()
+        {
+            string date = GetDateInput();
+            int quantity = GetNumberInput("Please insert quantity number (measure of your choice), or type 0 to return to menu");
+            
+            using (var connection = new SqliteConnection(connectionString))
+            {
+                connection.Open();
+                var tableCmd = connection.CreateCommand();
+                tableCmd.Parameters.AddWithValue("@date", date);
+                tableCmd.Parameters.AddWithValue("@quantity", quantity);
+
+                tableCmd.CommandText =
+                    $"INSERT INTO drinking_water(Date, Quantity) VALUES(@date, @quantity)";
+                
+                tableCmd.ExecuteNonQuery();
+                connection.Close();
+            }
+        }
+
+        void DeleteRecord()
+        {
+            Console.Clear();
+            GetAllRecords();
+            int id = GetNumberInput("Enter the ID of a record you want to delete, or type 0 to return to main menu");
+
+            if (id == 0)
+            {
+                GetUserInput();
+            }
+
+            using (var connection = new SqliteConnection(connectionString))
+            {
+                connection.Open();
+                var tableCmd = connection.CreateCommand();
+                tableCmd.Parameters.AddWithValue("@id", id);
+                
+                tableCmd.CommandText = $"DELETE FROM drinking_water WHERE Id = {@id}";
```

Comment:
🟠 **Mixed Parameters with String Interpolation**

💡 If you are adding parameters, don't mix in string interpolation into your query.

❌ `tableCmd.CommandText = $"DELETE FROM drinking_water WHERE Id = {@id}";`
✔️ `tableCmd.CommandText = "DELETE FROM drinking_water WHERE Id = @id";`